jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdConfiguration.java

```diff
@@ -59,11 +58,50 @@
      * @param clientSecret The client secret known only by the Client and the Authorization Server.
      */
     public OpenIdConfiguration(String provider, String clientId, String clientSecret)
+    {
+        this(provider, null, null, clientId, clientSecret);
+    }
+
+    /**
+     * Create an OpenID configuration for a specific OIDC provider.
+     * @param provider The URL of the OpenID provider.
+     * @param authorizationEndpoint the URL of the OpenID provider's authorization endpoint if configured.
+     * @param tokenEndpoint the URL of the OpenID provider's token endpoint if configured.
+     * @param clientId OAuth 2.0 Client Identifier valid at the Authorization Server.
+     * @param clientSecret The client secret known only by the Client and the Authorization Server.
+     */
+    public OpenIdConfiguration(String provider, String authorizationEndpoint, String tokenEndpoint, String clientId, String clientSecret)
```

Comment:
This extra ctor was added to try to maintain compatibility with existing `jetty-openid.xml`. Not sure if this is the way to do it and if it'll work. Seemed to from my testing, but insights from those more experienced would be apprecaited.

Reply:
This approach seems good to me, it keeps `jetty-openid.xml` simple.
It would be good to see some javadoc comment to explain that discovery is used if the `authorizationEndpoint` or `tokenEndpoint` aren't provided.